src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -147,7 +147,7 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
                     throw new InvalidOperationException("The monitored collection cannot be the same as the collection storing the leases.");
                 }
 
-                monitoredCosmosDBService = _configProvider.GetService(triggerConnectionString, resolvedPreferredLocations, attribute.UseDefaultJsonSerialization);
+                monitoredCosmosDBService = _configProvider.GetService(triggerConnectionString, resolvedPreferredLocations, false, attribute.UseDefaultJsonSerialization);
```

Comment:
What does this `false` do?

Reply:
This is a bug introduced on https://github.com/Azure/azure-webjobs-sdk-extensions/pull/575, basically `GetService` had originally 3 parameters, that PR added a 4th, but put it as the 3th.

![image](https://user-images.githubusercontent.com/1633401/94458811-512d8c00-016b-11eb-9dcc-628e01aadea4.png)

The false is the default value for `UseMultipleWriteLocations`.

Reply:
Ah, so we were inadvertently controlling `UseMultipleWriteLocations` with `UseDefaultJsonSerialization`? Got it, this makes sense now :-)